lib/dynamoid/transaction/create.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'action'
+
+module Dynamoid
+  module Transaction
+    class Create < Action
+
+      def run_callbacks
+        (yield; return) unless model
```

Comment:
I assume it's a branch for `create(User, attributes)`. Shouldn't we run callbacks as well in this case?

Reply:
Yes, this is done now.